pkg/registry/apis/folders/conversions.go

```diff
@@ -161,6 +157,8 @@ func convertToK8sResource(v *folder.Folder, namespacer request.NamespaceMapper)
 
 	meta.SetUpdatedTimestamp(&v.Updated)
 	if v.ID > 0 { // nolint:staticcheck
+		meta.SetDeprecatedInternalID(v.ID) // nolint:staticcheck
```

Comment:
Is this ID the folder ID or the dashboard.ID?

Reply:
This should be researched further, but I think it needs to be dashboard.ID - the folder id endpoint returns the dashboard.id

Reply:
At this point, this is the ID from the `dashboard` table. We are not even storing the `folder.ID` in unistore.

Makes sense?